<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#salir_2, #minimizar {
	border-radius: 10px;
	background-color: white;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
}

#salir_2:hover {
	background-color : red;
}

#minimizar:hover {
	background-color : #ADD8E6;
}

.QLabel {
	color: white;
}

#fAbajo{
background-color: #4a6eb0;
	border-color: rgb(0, 0, 0);
border-style: double;
border-radius: 14px;
}
</string>
  </property>
  <widget class="QPushButton" name="agregar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar paciente</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edad">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="apellido">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>551</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="busqueda">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="campoBuscar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
